aiida/engine/processes/calcjobs/manager.py

```diff
@@ -89,7 +108,11 @@ def _get_jobs_from_scheduler(self):
                 kwargs['jobs'] = self._get_jobs_with_scheduler()
 
             scheduler_response = scheduler.get_jobs(**kwargs)
+
+            # Update the last update time and clear the jobs cache
+            self._last_updated = time.time()
```

Comment:
just wondering: are there different standard ways of getting the time?
If not, it's fine - otherwise this might be something to standardize in aiida.common

Reply:
the `time.time` call just returns a float measured in seconds. Don't think it is worth to standardize this